jdi-light-angular-tests/src/test/java/io/github/epam/site/steps/States.java

```diff
@@ -0,0 +1,51 @@
+package io.github.epam.site.steps;
+
+import com.epam.jdi.light.elements.composite.WebPage;
+import io.github.com.StaticSite;
+import io.qameta.allure.Step;
+
+import static com.epam.jdi.light.elements.composite.WebPage.openUrl;
+import static com.epam.jdi.light.settings.WebSettings.DOMAIN;
+import static io.github.com.StaticSite.angPage;
+import static io.github.com.StaticSite.loginForm;
+import static io.github.com.StaticSite.logout;
+import static io.github.com.StaticSite.userIcon;
+import static io.github.com.StaticSite.userName;
+
+@SuppressWarnings("PMD.ClassNamingConventions")
```

Comment:
do we need it?

Reply:
Yes? the reason is the same as for TestInit class

Reply:
we could just suppress the specific warning in the codacy , instead of suppressing the whole class. so let's remove it from here and send me the link if codacy would show the warning, I'll turn off it

Reply:
Removed